actions/new.action.ts

```diff
@@ -203,7 +203,12 @@ const initializeGitRepository = async (dir: string) => {
 const createGitIgnoreFile = (dir: string, content?: string) => {
   const fileContent = content || defaultGitIgnore;
   const filePath = join(process.cwd(), dir, '.gitignore');
-  return promisify(fs.writeFile)(filePath, fileContent);
+  return fileExists(filePath).then(exists => {
+    if (!exists) {
+      return promisify(fs.writeFile)(filePath, fileContent);
+    }
+    return;
+  });
```

Comment:
why not use async/await here

```suggestion
  const gitIgnoreFileExists = await fileExists(filePath);
  if (!gitIgnoreFileExists) return promisify(fs.writeFile)(filePath, fileContent);
```